ChatQnA/benchmark/performance/helm_charts/README.md

```diff
@@ -20,7 +20,7 @@ HUGGINGFACEHUB_API_TOKEN: hf_xxxxx
 ```bash
 # Deploy a ChatQnA pipeline using the specified YAML configuration.
 # To deploy with different configurations, simply provide a different YAML file.
-helm install chatqna helm_charts/ -f customize.yaml
+python deployment.py --workflow=with_rerank --mode=tuned --num_nodes=4
```

Comment:
Let's list all the options for each parameter in the readme.

Reply:
fixed